frontend/packages/dev-console/src/components/taskruns/TaskRunDetails.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { SectionHeading, ResourceSummary } from '@console/internal/components/utils';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+
+export interface TaskRunDetailsProps {
+  obj: K8sResourceKind;
+}
+
+export const TaskRunDetails: React.FC<TaskRunDetailsProps> = ({ obj: taskRun }) => {
```

Comment:
Should use default export.

Any particular reason right now to make a custom details component? Won't the default do exactly the same thing?

Reply:
Yes, we need to pass the ```viewYamlComponent``` to ```editYaml``` for that I created this custom detail component. By default ```editYamlComponent``` is passed to ```editYAML```.

Reply:
@christianvogt I wonder if we want to consider extending the interface into optional "tabList" overrides... that way we don't really need to override the whole thing, just a portion.

Reply:
I think that's a reasonable change in the generic component as it still keeps it generic.